common/helpers.js

```diff
@@ -30,6 +30,10 @@ function validateDataObjectUrl(someUrl) {
     }
 }
 
+function isNullish(value) {
+    return value === null || typeof value === "undefined";
```

Comment:
Question, are we checking if something is `null` or `undefined` or are we checking if something is `null` or the string `"undefined"`.  The latter seems surprising (but entirely plausible). If it is the string, do you know what was causing that string?

Reply:
The former. It is checking whether it is `null` or `undefined`

Reply:
Since you are testing against `undefined` and not the string I don’t think you need that function isNullish. Instead can you try: https://stackoverflow.com/questions/2559318/how-to-check-for-an-undefined-or-null-variable-in-JavaScript I’m not certain though so please try and see what you see. Hope this helps

Reply:
Sorry about this. As I mentioned in Slack, I totally missed the `===` and the `typeof`. Thank you for adding the stackoverflow reference so future folks will understand why this was implemented this way!